boost.stacktrace does not build with Bazel

I just tried to build this module but it cannot build
```
# Configuration: fd810ba96138c7160862c6a2baa0d6e090abdca1e72b76a6bbf6220df0ae335e
# Execution platform: @@platforms//host:host

Use --sandbox_debug to see verbose messages from the sandbox and retain the sandbox build root for debugging
In file included from external/boost.stacktrace~/include/boost/stacktrace/frame.hpp:38,
                 from external/boost.stacktrace~/include/boost/stacktrace.hpp:15,
                 from dynamic_memory/main.cpp:1:
external/boost.stacktrace~/include/boost/stacktrace/safe_dump_to.hpp:212:21: fatal error: boost/stacktrace/detail/safe_dump_posix.ipp: No such file or directory
  212 | #           include <boost/stacktrace/detail/safe_dump_posix.ipp>
      |                     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
Target //dynamic_memory:main failed to build
```
The easiest way to reproduce is to create a dummy cpp file that includes `<boost/stacktrace.hpp>` and compile.

--- a/BUILD.bazel
+++ b/BUILD.bazel
@@ -7,6 +7,7 @@ cc_library(
     hdrs = glob([
         "include/**/*.hpp",
         "include/**/*.h",
+        "include/**/*.ipp,
     ]),
     includes = ["include"],
     deps = [

